uui/components/pickers/PickerInput.tsx

```diff
@@ -66,7 +73,12 @@ function PickerInputComponent<TItem, TId>({ highlightSearchMatches = true, ...pr
     };
 
     const renderTarget = (targetProps: IDropdownToggler & PickerTogglerProps<TItem, TId>) => {
-        const renderTargetFn = props.renderToggler || ((props) => <PickerToggler { ...props } />);
+        const renderTargetFn = props.renderToggler || ((pickerTogglerProps) => (
+            <PickerToggler
+                { ...pickerTogglerProps }
+                renderItem={ props.renderTag ? (renderItemProps) => props.renderTag(renderItemProps) : null }
```

Comment:
Can we add  `renderItem` into `targetProps` params. And also why do we have this condition? i guess we can just pass it as is, without wrapping into arrow function.

Reply:
fixed